package.yaml

```diff
@@ -1,5 +1,5 @@
 name: purescript
-version: '0.14.0-rc5' # note: when updating this, update the prerelease identifier in app/Version.hs too!
```

Comment:
Can we leave this comment in please, for next time we make a prerelease version?